index.html

```diff
@@ -1,12 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Beginning Web Development</title>
     <link rel="stylesheet" href="main.css">
 </head>
+
 <body>
 
-<script src="main.js"></script>
+    <!-- NAV BAR (this is to test using github/pull requests)-->
+
+    <!-- HERO -->
+
+    <!-- ABOUT -->
+
+    <!-- BLOG -->
+
+    <!-- FOOTER -->
+
+    <footer>
+        <div class="row">
+            <div class="col col--w30">
+                <a href="#" target="_blank"><img src="./images/bwd-logo-darkmode.svg" alt="BWD logo"></a>
+            </div>
+
+            <div class="col col--w30">
+                <ul>
+                    <li class="list-heading">Quick Links:</li>
+                    <li><a target="_blank" href="https://www.meetup.com/beginning-web-development/">MeetUp Page</a></li>
+                    <li><a target="_blank" href="https://discord.gg/WfeGwP6y64">Discord</a></li>
+                </ul>
+            </div>
+
+            <div class="col col--w30">
+                <ul>
+                    <li class="list-heading">Resources:</li>
+                    <li><a target="_blank" href="https://freecodecamp.org/">FreeCodeCamp.org</a>
+                    </li>
+                    <li><a target="_blank" href="https://www.theodinproject.com/">OdinProject.com</a></li>
```

Comment:
I'd recommend adding the 'rel="noopener noreferrer"' attribute to external links for increased security.

Reply:
Seems like there's no need for it anymore. According to the following, many major browsers have implemented noopener and noreferrer for _blank targets to external links since 2021: https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

On the other hand, I had no idea that this was even a thing. Was definitely educational! Thanks for pointing it out!

Reply:
Thanks so much for this link and information! This thing was annoying me a lot since nothing really had a clear answer on it. It definitely seems like something people are only using out of habit nowadays. Regardless I wouldn't see why browsers wouldn't already have this attack patched. And you're welcome.